woof-code/rootfs-skeleton/usr/sbin/puppyinstaller

```diff
@@ -864,7 +865,7 @@ case $PMEDIA in usb*)
 		yaf-splash -bg orange -text "`eval_gettext \"Please wait, copying files to \\\$DESTPART ...\"`" &
 		XPID=$!
 		mkdir -p /mnt/data
-		for i in vmlinuz initrd.gz ${DISTRO_PUPPYSFS} ${DISTRO_ZDRVSFS} ${DISTRO_FDRVSFS} ${DISTRO_ADRVSFS} ${DISTRO_YDRVSFS}; do
+		for i in vmlinuz initrd.gz ${DISTRO_PUPPYSFS} ${DISTRO_ZDRVSFS} ${DISTRO_FDRVSFS} ${DISTRO_ADRVSFS} ${DISTRO_YDRVSFS} ${DISTRO_BDRVSFS}; do
```

Comment:
Why not align this list with https://github.com/puppylinux-woof-CE/woof-CE/blob/5ff59258c353b2f8739af7099b9b4ebd7f34b6f7/woof-code/rootfs-skeleton/usr/sbin/bootflash#L873?

Reply:
Full installs are deprecated and we prefer to promote frugal installs - however this addition does seem sensible.

There is no need to update the historic Copyright comment as Github records all "blame".

Reply:
patch updated to reverse copyright change.

Reply:
But you haven't added kbuild and other SFSs 🤷🏿